Sources/Orbit/Support/Components/TimelineStep.swift

```diff
@@ -21,14 +21,20 @@ public extension VerticalAlignment {
 ///   - ``Timeline``
 ///
 /// - Note: [Orbit definition](https://orbit.kiwi/components/progress-indicators/timeline/)
-public struct TimelineStep<Header: View, Content: View>: View {
+public struct TimelineStep<Header: View, Content: View, Footer: View>: View {
 
     @Environment(\.sizeCategory) var sizeCategory
+    @Environment(\.accessibilityReduceMotion) var reduceMotion
+
+    let animationSpeed: CGFloat = 0.25
 
     let style: TimelineStepStyle
     let isIconFirstTextLineCentered: Bool
     @ViewBuilder let header: Header
```

Comment:
With the new design, maybe we do not need a customizable header anymore and the only customizable content could be the footer.

Reply:
Removed

Reply:
Nice. Now the `TimelineItem where Footer == EmptyView` init should reuse the main init.